### PR TITLE
Added parameter to choose the number of samples per ray

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: livre/Eq/Settings/RenderSettings.cpp [hunk 1]
```diff
@@ -68,12 +65,6 @@ void RenderSettings::setRendererType( const RendererType rendererType )
     setDirty( DIRTY_RENDERER_TYPE );
 }
 
-void RenderSettings::setNumberOfSamples( const uint32_t nbOfSamples )
-{
-    nSamples_ = nbOfSamples;
-    setDirty( DIRTY_SAMPLES );
```
Comment (tribal-tec) [inline livre/Eq/Settings/RenderSettings.cpp]: DIRTY_SAMPLES  is now unused, remove it

File: livre/Lib/Configuration/VolumeRendererParameters.cpp [hunk 2]
```diff
@@ -29,6 +29,7 @@ const std::string TEXTURECACHEMEM_PARAM = "texture-cache-mem";
 const std::string TEXTUREDATACACHEMEM_PARAM = "texturedata-cache-mem";
 const std::string MINLOD_PARAM = "min-lod";
 const std::string MAXLOD_PARAM = "max-lod";
+const std::string SAMPLESPERRAY_PARAM = "samples-per-ray";
```
Comment (tribal-tec) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: call it just 'samples'? Personally, I find this too long

Comment (rdumusc) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: I disagree here, samples is really not clear enough. ;-) This is an advanced option anyway, not something you are likely to use everyday

Comment (chevtche) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: I don't agree... samples alone means nothing. For example we will have most probably samples-per-pixel in the future. I also think samples-per-ray explain very well what it is, no error possible.

File: livre/Eq/Settings/RenderSettings.cpp (hunk 1 above)
Comment (chevtche) [inline livre/Eq/Settings/RenderSettings.cpp]: Updated

**Comment by eile** [comment]
+1

**Comment by chevtche** [comment]
Retest this please

**Pull request merged**